docs/learning/quantity/index.md

```diff
@@ -229,3 +231,62 @@ The quantity formatter supports parsing mathematical operations. The operation i
 ```
 
 </details>
+
+## Persistence
+
+We expose APIs and interfaces to support persistence of formats. Different from [KindOfQuantity](../../bis/ec/kindofquantity.md), which enables persistence of formats at the schema level, this section covers persistence at the application level.
+
+### PresentationFormatSet
+
+[PresentationFormatSet]($quantity) defines properties necessary to support persistence of a set of formats.
+
+Formats defined in a PFS require a `id` field.
+
+When a format's `id` matches a [schema Format's](../../bis/ec/ec-format.md#format) `typeName`, the PFS's format should override that schema format temporarily, avoiding making changes on a schema level.
+
+<details>
+<summary>Example of a Presentation Format Set in TypeScript</summary>
+
+```Typescript
+{
+  id: "metric",
+  name: "Metric",
+  // TODO: Should the unit system be included? Then, the id and name above doesn't have to conform to a unit system.
+  unitSystem: "metric",
+  formats: [ //  TODO: Array, or <key,FormatProps> to preserve uniqueness?
+    {
+      id: "Length", // Should there be more meaning/rules to the Id? If Koq, then should be <Schema>.<Name>
```

Comment:
@diegoalexdiaz should these be tied to a KoQ's fullName?

Reply:
I'm catching up with the implementation of _PresentationFormatSet_ so I'm not aware of all its details. However, its description sounds like it came from the perception that the concept of a KoQ only applies to cases defined in a schema so some parallel approach was needed for the cases where no schema is involved.

IMO, that perception is incorrect. I believe the Civil Review Tools did just that - create an in-memory KoQ by code associated to the formatting needed.

Reply:
I agree we should be looking at the possibility of using KOQ for everything instead of having two sources of keys.  Having an undescribed id field is confusing at both levels, I'd prefer we don't use the name 'id' if a more descriptive name could be used, if not then we should describe what the id means.

Reply:
I've updated the `formats` from an array to a key,value map, and updated the documentation to describe we are expecting a valid ECName for the map's key. That's just the expectation though, in reality anyone can put anything as the name, but of course that format won't be picked up by tools that use formatting, so it functionally wouldn't matter